CHANGELOG.md

```diff
@@ -10,6 +10,31 @@ and this project adheres to
 
 ---
 
+## [0.24.0] - 2025-05-30
+
+### Added
+
+- Cache keys are now dependent on implementation (Redis, DynamoDB, InMemory)
+  - Redis cache keys now use a format of
+    `{service_name}:auth0rs_tokens:{client_id}:{token_version}:{audience}` i.e.
+    the microservice name using the bridge is prepended, this should help with
+    permission handling
+  - DynamoDb and InMemory use a format of
```

Comment:
InMemory is an implementation detail that's not really observable by callers, I wouldn't even mention it